src/helpers/_visually-hidden.scss

```diff
@@ -9,26 +9,28 @@
 /// @link https://github.com/h5bp/html5-boilerplate/blob/9f13695d21ff92c55c78dfa9f16bb02a1b6e911f/src/css/main.css#L121-L158
 ///   - h5bp/html5-boilerplate - Thanks!
 ///
+/// @param {Boolean} $important [false] - Whether to mark as `!important`
+///
 /// @access public
 
-@mixin govuk-visually-hidden {
-  position: absolute;
+@mixin govuk-visually-hidden($important: false) {
```

Comment:
I think because this is defaulted to false, it will not solve the original issue.

We need to make also consider if this is a breaking change.

1. Does it change the functionality of the components that are using this mixin
2. Could it change the functionality of our users's components in a way that changes the behaviour
3. Is it safe to enable this by default for the helper classes

I do not think that having this on by default could break any of these scenarios but would be worth checking the components that use these mixins to be sure.

- https://github.com/alphagov/govuk-frontend/search?q=govuk-visually-hidden&unscoped_q=govuk-visually-hidden
- https://github.com/alphagov/govuk-frontend/search?q=govuk-visually-hidden-focusable&unscoped_q=govuk-visually-hidden-focusable